shared/src/test/kotlin/com/egm/stellio/shared/util/MockkedHandler.kt

```diff
@@ -3,14 +3,32 @@ package com.egm.stellio.shared.util
 import org.springframework.http.HttpStatus
 import org.springframework.http.MediaType
 import org.springframework.http.ResponseEntity
-import org.springframework.web.bind.annotation.PostMapping
-import org.springframework.web.bind.annotation.RequestMapping
-import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.bind.annotation.*
+import reactor.core.publisher.Mono
+import com.egm.stellio.shared.web.ExceptionHandler as customExceptionHandler
 
 @RestController
 @RequestMapping("/router/mockkedroute")
-class MockkedHandler {
+class MockkedHandler : customExceptionHandler() {
 
     @PostMapping(consumes = [MediaType.APPLICATION_JSON_VALUE, JSON_LD_CONTENT_TYPE])
     fun post() = ResponseEntity.status(HttpStatus.CREATED).build<String>()
+
+    @PostMapping("/validate-json-ld")
```

Comment:
you don't really need both endpoints, no?

Reply:
Indeed, I updated the MockedHandlerTests [here](https://github.com/stellio-hub/stellio-context-broker/pull/220/commits/71d454d905a7d48469e1cfa1f98f6d3871f75b73)